### PR TITLE
Edited button definition and calls

### PR DESCRIPTION
**Pull request by agalbrai**
Changed some button definitions to match the BCGov style guide and changed some button calls  for more visual consistency

### CODE REVIEW

File: frontend/src/buttons.scss [hunk 1]
```diff
@@ -37,9 +37,10 @@
   height: 44px;
 }
 
+/*
 .btn.btn-warning {
```
Comment (Fosol) [inline frontend/src/buttons.scss:41]: Rip it out if we don't need it.

File: frontend/src/buttons.scss [hunk 2]
```diff
@@ -54,17 +55,31 @@
...
+  cursor: pointer;
 }
 
-.btn:hover {
+.btn.btn-primary:hover {
+  text-decoration: underline;
   opacity: 0.8;
 }
+
+/*
```
Comment (Fosol) [inline frontend/src/buttons.scss:77]: Rip it out if we don't need it.

File: frontend/src/buttons.scss [hunk 3]
```diff
@@ -75,20 +90,27 @@
...
-  background-color: #ffffff;
-  font-family: 'BCSans', Fallback, sans-serif;
-  font-size: 16px;
-  color: $secondary-variant-color;
-  text-decoration: none solid $secondary-variant-color;
+.btn.btn.btn-secondary {
+  background: none;
+  border-radius: 4px;
+  border: 1px solid $primary-color;
+  //padding: 10px 30px;
```
Comment (Fosol) [inline frontend/src/buttons.scss:97]: Rip it out if we don't need it.

File: frontend/src/buttons.scss [hunk 4]
```diff
@@ -75,20 +90,27 @@
...
   text-align: center;
-  border-radius: 0;
+  text-decoration: none;
+  //display: block;
```
Comment (Fosol) [inline frontend/src/buttons.scss:100]: Rip it out if we don't need it.

File: frontend/src/buttons.scss [hunk 5]
```diff
@@ -127,27 +149,37 @@
...
 
-.btn.btn-warning {
+.btn.btn.btn-warning {
   background-color: $accent-color;
-  font-family: 'BCSans', Fallback, sans-serif;
-  text-decoration: none solid rgb(255, 255, 255);
+  border: 1px solid $accent-color;
+  border-radius: 4px;
   color: #ffffff;
+  //padding: 12px 32px;
```
Comment (Fosol) [inline frontend/src/buttons.scss:174]: Rip it out if we don't need it.

File: frontend/src/features/admin/access/components/Details.tsx [hunk 6]
```diff
@@ -84,7 +84,7 @@ export const AccessRequestDetails: React.FC<IProps> = ({ request, onClose }) =>
         </Modal.Body>
 
         <Modal.Footer>
-          <Button variant="warning" onClick={onClose}>
+          <Button variant="primary" onClick={onClose}>
```
Comment (Fosol) [inline frontend/src/features/admin/access/components/Details.tsx:87]: Don't change this, but would be nice to extract and create a shared Close button component.

File: frontend/src/features/admin/edit-user/EditUserPage.tsx [hunk 7]
```diff
@@ -220,7 +220,7 @@ const EditUserPage = (props: IEditUserPageProps) => {
 
                 <Row className="justify-content-md-center">
                   <ButtonToolbar className="cancelSave">
-                    <Button className="mr-5" variant="outline-dark" type="button" onClick={goBack}>
+                    <Button className="mr-5" variant="secondary" type="button" onClick={goBack}>
```
Comment (Fosol) [inline frontend/src/features/admin/edit-user/EditUserPage.tsx:223]: Would be nice to extract this button too as a shared component

**Review by Fosol — APPROVED** [review]
Thanks!

File: frontend/src/features/admin/access/components/Details.tsx (hunk 6 above)
Comment (agalbrai) [reply frontend/src/features/admin/access/components/Details.tsx:87]: So leave it as 'primary' or change it back to 'warning'? components are a bit outside of my realm of understanding at the moment

**Review by goemen — APPROVED** [review]

**Pull request merged**